src/main/scala/wdl4s/values/WdlFile.scala

```diff
@@ -35,14 +39,20 @@ sealed trait WdlFile extends WdlPrimitive {
     case _ => invalid(s"$value == $rhs")
   }
 
-  override def valueString = value.toString
+  override def valueString: String = value.toString
 }
 
-case class WdlSingleFile(value: String) extends WdlFile {
-  override def toWdlString = "\"" + value.toString + "\""
+case class WdlSingleFile(origValue: String) extends WdlFile {
+  override def toWdlString: String = "\"" + value.toString + "\""
+
+  // Validate file path
```

Comment:
I think in this case the `origValue` could become refined, e.g. `case class WdlSingleFile(origValue: String Refined UrI)`